webview/src/desktopMain/kotlin/com/multiplatform/webview/web/DesktopWebView.kt

```diff
@@ -27,36 +26,33 @@ class DesktopWebView(private val webView: WebView) : IWebView {
         encoding: String?,
         historyUrl: String?
     ) {
-        Platform.runLater {
-            engine.loadContent(html)
-        }
+        webView.loadURL("data:text/html;charset=utf-8,$html")
     }
 
-    override fun postUrl(url: String, postData: ByteArray) = Platform.runLater {
-        engine.loadContent(postData.toString(), "text/html")
+    override fun postUrl(url: String, postData: ByteArray) {
+        val request = CefRequest.create().apply {
+            this.url = url
+            this.postData = CefPostData.create().apply {
+                this.addElement(CefPostDataElement.create().apply {
+                    this.setToBytes(postData.size, postData)
+                })
+            }
+        }
+        webView.loadRequest(request)
     }
 
-    override fun goBack() = Platform.runLater {
-        engine.goBack()
-    }
+    override fun goBack() = webView.goBack()
 
-    override fun goForward() = Platform.runLater {
-        engine.goForward()
-    }
+    override fun goForward() = webView.goForward()
 
-    override fun reload() = Platform.runLater {
-        engine.reload()
-    }
+    override fun reload() = webView.reload()
 
-    override fun stopLoading() = Platform.runLater {
-        engine.stopLoading()
-    }
+    override fun stopLoading() = webView.stopLoad()
 
-    override fun evaluateJavaScript(script: String, callback: ((String) -> Unit)?) = Platform.runLater {
+    override fun evaluateJavaScript(script: String, callback: ((String) -> Unit)?) {
         Logger.i {
             "evaluateJavaScript: $script"
         }
-        val res = engine.executeScript(script)
-        callback?.invoke(res.toString())
+        // ToDo("implement for jcef")
```

Comment:
Does jcef support Javascript evaluation?

Reply:
Currently not. It can execute JavaScript but without getting a response.

I already requested a change (since it's possible with CefSharp) but idk how long this will take.

Reply:
Maybe we can change the `evaluateJavaScript` to platform specific code for now.
So this can be called on Android and iOS only, and Desktop get's a `executeJavascript` method until https://github.com/chromiumembedded/java-cef/issues/450 is fixed

This PR brings a breaking change anyway, since the desktop module needs to be configured.

Reply:
Or we keep the common evaluation and always return null.
But this may misleading and needs proper documentation.

Either way I'd still switch to the CEF browser, not just because of my mentioned issues but other things as well.

- It's much faster
- It's much smoother
- Does not depend on JavaFX
	- This is especially bad because Compose Desktop uses Swing and therefor swing coroutines are used, but if we add the javafx coroutines as dependency it may break other stuff as well

Reply:
### Just collecting some ideas

We could also provide a `canEvaluateJavascript(): Boolean` method.
Then the user has to check wether he wan't to call it.

Reply:
How about we just execute JavaScript but not give the response? I think it is better than just not supporting it since most JavaScript calls do not need a response. We can write a clear comment on the method and explain the reason. I think the user can understand. What do you think? Thanks for your contribution again!